packages/console/src/components/TextInput/index.tsx

```diff
@@ -77,7 +66,9 @@ function TextInput(
             className: classNames([suffix.props.className, styles.suffix]),
           })}
       </div>
-      {hasError && errorMessage && <div className={styles.errorMessage}>{errorMessage}</div>}
+      {Boolean(error) && typeof error === 'string' && (
```

Comment:
1. We treat the empty string as no error since there is no meaning when an error is an empty string.
2. The error content can not be limited to TFuncKey, since the error message usually needs to be constructed ahead.